fitbenchmarking/results_processing/tests/test_tables.py

```diff
@@ -60,7 +60,8 @@ def setUp(self):
 
         self.fig_dir = os.path.join(root, 'results_processing',
                                     'tests', 'figures')
-        os.mkdir(self.fig_dir)
+        if not os.path.exists(self.fig_dir):
```

Comment:
This is a better why of doing it from a developer perspective. It would not make a difference if the tests run correctly but it makes it easier if you are using the debug mode and stop the debugger without a proper test teardown.